gen/calc.py

```diff
@@ -296,7 +366,18 @@ def calculate_exhibitor_static_ensemble(master_list):
         'ui_banner_header_content': 'null',
         'ui_banner_footer_content': 'null',
         'ui_banner_image_path': 'null',
-        'ui_banner_dismissible': 'null'
+        'ui_banner_dismissible': 'null',
+        'dcos_overlay_network': '{                    \
```

Comment:
I echo @cmaloney and would add that it might be nice to have the parameters configurable, that way in the future we could configure subsets of the defaulted JSON (though this is purely an armchair observation, it would make validating the contents of the config more attainable as well if we had a complete data structure and not a string blob).